client/layout/index.jsx

```diff
@@ -134,7 +137,8 @@ Layout = React.createClass( {
 				{ 'is-support-user': this.props.isSupportUser },
 				{ 'has-no-sidebar': ! this.props.hasSidebar },
 				{ 'wp-singletree-layout': !! this.props.primary },
-				{ 'has-chat': this.props.chatIsOpen }
+				{ 'has-chat': this.props.chatIsOpen },
+				`is-${ this.props.colorSchemePreference }-theme`,
```

Comment:
I see some other classnames (` `is-group-${ this.props.section.group }` `) are approached in the same way here but this makes me wonder if there's a chance that this could come up `is-undefined-theme`. Do you think a default prop value would make sense or are we happy that there will always be a prop value coming from redux?

Reply:
In `DEFAULT_PREFERENCE_VALUES`we provide `'default'` as the default theme. So usually there should always be a value provided. In addition if there technically ever was a case where the value was undefined, it would not have an impact. Those classes are just an override to the default styles. The class would simply get ignored and the default theme displayed. So even if we would add an additional check here, there wouldn't be any difference in the result. But as always happy to change it if that would help the cause :)

Reply:
The default in `DEFAULT_PREFERENCE_VALUES` sounds like it'll suffice :) Thanks for clearing that up @frontdevde !